book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+end
```

Comment:
To fulfill the `many to many` relationship  between rentals and books, a method to add rentals should be present in the book class😉.
<img width="482" alt="image" src="https://github.com/Jawad-Nazari/School_library/assets/64643995/77ba1742-67cb-4efa-a4c7-127d591b4c3c">

you can checkout[ this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for a reminder